docs/_templates/sidebarlogo.html

```diff
@@ -50,13 +50,14 @@ <h3>Other Projects</h3>
   <li><a href="https://pipenv-pipes.readthedocs.io/en/latest/">Pipenv-Pipes</a></li>
 </ul>
 
-<p>More projects founded by <a href="http://kennethreitz.org/">Kenneth Reitz</a>:</p>
+<p>More projects founded by <a href="https://kenreitz.org">Kenneth Reitz</a>:</p>
 <ul>
-    <li><a href="http://pep8.org/">pep8.org</a></li>
-    <li><a href="http://httpbin.org/">httpbin.org</a></li>
+    <li><a href="https://pep8.org/">pep8.org</a></li>
+    <li><a href="https://httpbin.org/">httpbin.org</a></li>
     <li><a href="https://howtopython.org">HowToPython.org</a></li>
-    <li><a href="http://python-guide.org">The Python Guide</a></li>
-    <li><a href="http://python-requests.org/">Requests: HTTP for Humans</a></li>
+    <li><a href="https://python-guide.org">The Python Guide</a></li>
+    <li><a href="https://requests.readthedocs.io">Requests: HTTP for Humans</a></li>
+    <li><a href="https://github.com/kennethreitz/maya">Maya: Datetimes for Humans</a></li>
```

Comment:
Maya has transferred ownership. Also, this wasn’t present in master?

Reply:
I've updated the PR with its new location - and same with `records` as well.  This change isn't present in `master` - the latest version of the docs (https://pipenv.pypa.io/en/latest/) has the old location.